src/main/java/org/prebid/server/handler/OptoutHandler.java

```diff
@@ -41,6 +44,13 @@ public OptoutHandler(GoogleRecaptchaVerifier googleRecaptchaVerifier, UidsCookie
         this.optinUrl = Objects.requireNonNull(optinUrl);
     }
 
+    @Override
+    public List<HttpEndpoint> endpoints() {
+        return List.of(
+                HttpEndpoint.of(HttpMethod.GET, "/optout"),
+                HttpEndpoint.of(HttpMethod.POST, "/optout"));
+    }
```

Comment:
I don't like the idea that handler, can have more then one endpoint.
It makes a thought that single responsibility principle is broken.
Maybe it is worth to separate for two different? And make ApllicationResource class have only `endpoint` method?